_data/team-academic.yml

```
@@ -27,7 +27,7 @@
 - name: "Silvana"
   surname: "Camboim"
   title: "Universidade Federal do Paraná"
-  thumbnailUrl: "generic.jpg"
+  thumbnailUrl: "Silvana_Camboim.jpg"
```

Please keep image name consistent: `Firstname Lastname.extension`
```suggestion
  thumbnailUrl: "Silvana Camboim.jpg"
```

---

updated